stl/src/syserror_import_lib.cpp

```diff
@@ -51,11 +51,12 @@ _NODISCARD size_t __CLRCALL_PURE_OR_STDCALL __std_system_error_allocate_message(
     if (_Ret == 0) {
         _Lang_id = 0;
     }
+    // Flag FORMAT_MESSAGE_MAX_WIDTH_MASK Removes All Trailing \r \n Leaves Only Whitespace Character
     const unsigned long _Chars =
-        FormatMessageA(FORMAT_MESSAGE_ALLOCATE_BUFFER | FORMAT_MESSAGE_FROM_SYSTEM | FORMAT_MESSAGE_IGNORE_INSERTS,
+        FormatMessageA(FORMAT_MESSAGE_ALLOCATE_BUFFER | FORMAT_MESSAGE_FROM_SYSTEM | FORMAT_MESSAGE_IGNORE_INSERTS | FORMAT_MESSAGE_MAX_WIDTH_MASK,

```

Comment:
too long line.
Just run `clang-format -i stl/src/syserror_import_lib.cpp` 
or 
you can run ` cmake --build <builddir> --target format` for formatting every file.

Reply:
Formated Line